packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -4,7 +4,7 @@
         tab: null,
 
         init: function () {
-            this.tab = this.getTabs()[@js($getActiveTab() - 1)]
+            this.tab = this.getTabs()[@js($getActiveTab()) - 1]
```

Comment:
Lol I wasn't sure about the preferred way. 😂

Reply:
im just being picky lol